analyzers/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -183,7 +183,7 @@ public void Cfg_ExtremelyNestedExpression_NotSupported_FromExpression()
             Action a = () => CSharpControlFlowGraph.Create(equalsValueSyntax.Value, semanticModel);
 
             a.Should().Throw<NotSupportedException>().WithMessage("Too complex expression");
-            CSharpControlFlowGraph.TryGet(method.Body, semanticModel, out _).Should().BeFalse();
+            CSharpControlFlowGraph.TryGet(equalsValueSyntax.Value, semanticModel, out _).Should().BeFalse();
```

Comment:
Fixing old bug, `equalsValueSyntax.Value` was intended here, just as in the `Create(..)` above.